components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/gen/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/dto/InputTypeEnum.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.rest.api.server.claim.management.v1.dto;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Enum representing the input types for user attributes in a UI form.
+ */
+public enum InputTypeEnum {
+
+    SELECT("select"),
+    RADIO_BUTTON("radio_button"),
+
+    MULTI_SELECT("multi_select"),
+    CHECKBOX("checkbox"),
+
+    TEXT_FIELD("text_field"),
+    TEXT_AREA("text_area"),
+
+    DATE_PICKER("date_picker"),
+    NUMBER_PICKER("number_picker"),
+
+    TOGGLE("toggle");
```

Comment:
Need to extend, and refactor names with properly align with the UI widgets used in the popular UI libraries 
(Ex: React)